lib/mix/tasks/compile.elixir_make.ex

```diff
@@ -56,13 +56,19 @@ defmodule Mix.Tasks.Compile.ElixirMake do
 
   The following options configure precompilation:
 
-    * `:make_precompiler` - the precompiled module to use. Defaults to none.
-
-    * `:make_precompiled_url` - the download URL template. Defaults to none.
+    * `:make_precompiler` - a two-element tuple with the precompiled type
+      and module to use. The precompile type is either `:nif` or `:port`
+      and then the precompilation module. If the type is a `:nif`, it looks
+      for a DDL or a shared object as precompilation target given by
```

Comment:
Sorry found a typo here :)

```suggestion
      for a DLL or a shared object as precompilation target given by
```